components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -408,6 +408,9 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     protected async doUpdateUser(): Promise<void> {
         // execute the check for the setup to be shown until the setup is not required.
         // cf. evaluation of the condition in `checkUser`
+        if (!this.config.showSetupModal) {
+            this.showSetupCondition = { value: false };
```

Comment:
I see, the reason was, that `getUserCount` didn't include the admin account.

Reply:
Jup, and it needs to stay that way to keep self-hosted setup working - until we remove it.